src/BleError.js

```diff
@@ -42,17 +42,17 @@ export class BleError extends Error {
    */
   reason: ?string
 
-  constructor(nativeBleError: NativeBleError | string) {
+  constructor(nativeBleError: NativeBleError | string, errorMessagesArray: Array<string>) {
```

Comment:
Array is not a good match here. I think that you should create exported type, which will tel user more about how it should be constructed:

in BleError:
```js
export type BleErrorCodeMessageType = {[$Values<typeof BleErrorCode>]: string}

export const BleErrorCodeMessage: BleErrorCodeMessageType = {
```

in above function:
```js
constructor(nativeBleError: NativeBleError | string, errorMessageDict: BleErrorCodeMessageType) {
```